src/backend/distributed/sql/udfs/compress_old_partitions/latest.sql

```diff
@@ -0,0 +1,13 @@
+CREATE OR REPLACE PROCEDURE pg_catalog.compress_old_partitions(
+  table_name regclass,
+  older_than timestamptz)
+LANGUAGE plpgsql
+AS $$
+BEGIN
+    CALL alter_old_partitions_set_access_method(table_name, older_than, 'columnar');
+ END;
+$$;
+COMMENT ON PROCEDURE pg_catalog.compress_old_partitions(
+  table_name regclass,
+  older_than timestamptz)
+IS 'compress old partitions of a time-partitioned table';
```

Comment:
this seems to be unrelated to the UDF you are introducing here, or am I completely missing how this is related to `citus_get_dependencies_for_object`